plug-ins/dns_spoof/dns_spoof.c

```diff
@@ -490,6 +506,18 @@ static void dns_spoof(struct packet_object *po)
 }
 
 /*
+ * Converts little-endian unsigned 32-bit integer to big-endian unsigned 32-bit integer
+ */
+static u_int32 little_to_big(u_int32 n)
+{
+	/* Convert TTL to big-endian */
+	return ((n>>24)&0x000000ff) | // move byte 3 to byte 0
+					 ((n<<8)&0x00ff0000) | // move byte 1 to byte 2
+					 ((n>>8)&0x0000ff00) | // move byte 2 to byte 1
+					 ((n<<24)&0xff000000); // byte 0 to byte 3
+}
+
```

Comment:
What is this *little_to_big()* function about? 
Usually you just have to convert host-ordered bytes to network-ordered bytes using the standard function **htonl()**.